src/pointer-analysis/dereference.cpp

```diff
@@ -1262,7 +1262,7 @@ void dereferencet::construct_from_const_struct_offset(
       return;
     }
 
-    if(int_offset < (m_offs + m_size))
+    if(int_offset < m_offs + m_size)
```

Comment:
I would prefer the previous version for the readability's sake :-)

Reply:
strange :) reverted.